docs/flow/config-language/expressions/function_calls.md

```diff
@@ -6,3 +6,25 @@ weight: 400
 ---
 
 # Function Calls
+Function calls is one more River feature that allows users to build richer
+expresisons.
+
+Functions take zero or more arguments as their input and always return a single
+value as their output. Functions cannot be constructed by users, but can be
+either called from River's standard library, or exported by a compoonent.
+
+## Standard library functions
+River contains a [standard library][] of useful functions. Some enable
+interaction with the host system (eg. reading from an environment variable), or
+allow for more complex expressions (eg. concatenating arrays or decoding JSON
+strings into objects).
+```river
+env("HOME")
+json_decode(local.file.cfg.contents)["namespace"]
+```
```

Comment:
The standard library is pretty barren for now. Should we mention that we're looking forward to expand as we identify new use cases, or what would be empower users?

Reply:
We say exactly this in docs/flow/reference/stdlib/_index.md: 

https://github.com/grafana/agent/blob/cbbfabb8b7844d702409dc62dcc32fd330973ec5/docs/flow/reference/stdlib/_index.md?plain=1#L10-L13